Gems/ScriptCanvas/Code/Editor/View/Widgets/SourceHandlePropertyAssetCtrl.cpp

```diff
@@ -18,6 +18,8 @@
 
 namespace ScriptCanvasEditor
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether